rdt-app/src/main/java/io/ona/rdt_app/util/RDTJsonFormUtils.java

```diff
@@ -82,17 +84,16 @@ protected ProfileImage doInBackground(Void... voids) {
                         profileImage.setSyncStatus(ImageRepository.TYPE_Unsynced);
                         ImageRepository imageRepo = RDTApplication.getInstance().getContext().imageRepository();
                         imageRepo.add(profileImage);
-
-//                        saveImageToGallery(context, image); // todo: only enable this in debug apks
+                        // saveImageToGallery(context, image);
                     }
                 } catch (FileNotFoundException e) {
-                    Log.e(TAG, "Failed to save static image to disk");
+                    Log.e(TAG, e.getStackTrace().toString());
```

Comment:
Consider to use Timber and log the error directly instead of stackTrace as string

Reply:
Done.